implementation/src/main/java/io/smallrye/faulttolerance/FaultToleranceMetricsPublisher.java

```diff
@@ -0,0 +1,34 @@
+package io.smallrye.faulttolerance;
+
+import com.netflix.hystrix.*;
+import com.netflix.hystrix.strategy.metrics.HystrixMetricsPublisher;
+import com.netflix.hystrix.strategy.metrics.HystrixMetricsPublisherCollapser;
+import com.netflix.hystrix.strategy.metrics.HystrixMetricsPublisherCommand;
+import com.netflix.hystrix.strategy.metrics.HystrixMetricsPublisherThreadPool;
+
+public class FaultToleranceMetricsPublisher extends HystrixMetricsPublisher {
+
+
+    @Override
+    public HystrixMetricsPublisherCommand getMetricsPublisherForCommand(HystrixCommandKey commandKey, HystrixCommandGroupKey commandGroupKey, HystrixCommandMetrics metrics, HystrixCircuitBreaker circuitBreaker, HystrixCommandProperties properties) {
+        System.out.println("**** Here ****");
```

Comment:
Remove or maybe some more meaningful message? Also we use `jboss-logging` in `smallrye-fault-tolerance`...

Reply:
File doesn't exist anymore